shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -127,6 +127,42 @@ TEST_F(AccessibilityBridgeTest, DeletesChildrenTransitively) {
   EXPECT_FALSE(semantics_manager_.UpdateOverflowed());
 }
 
+TEST_F(AccessibilityBridgeTest, PopulatesStates) {
+  flutter::SemanticsNode node0;
+  node0.id = 0u;
+  // HasCheckedState = true
+  // IsChecked = false
+  // IsSelected = false
+  // IsHidden = true
+  node0.flags = 0u;
```

Comment:
This is actually setting those flags.  You can see the enum definition in `semantics_node.h`.